web/src/components/organisms/Asset/Asset/hooks.ts

```diff
@@ -0,0 +1,81 @@
+import { useCallback, useEffect, useState } from "react";
+
+import { Asset } from "@reearth-cms/components/molecules/Asset/asset.type";
+import { viewerRef } from "@reearth-cms/components/molecules/Asset/Asset/AssetBody/index";
+import {
+  PreviewType,
+  useGetAssetQuery,
+  useUpdateAssetMutation,
+} from "@reearth-cms/gql/graphql-client-api";
+
+export default (assetId?: string) => {
+  const [asset, setAsset] = useState<Asset>({} as Asset);
```

Comment:
If this works I think its better
```suggestion
  const [asset, setAsset] = useState<Asset | {}>({});
```

Reply:
I got some errors after doing this suggested change.

Reply:
Ok!